examples/storm-hdfs-examples/src/main/java/org/apache/storm/hdfs/bolt/SequenceFileTopology.java

```diff
@@ -151,15 +145,15 @@ public void nextTuple() {
             }
             count++;
             total++;
-            if(count > 20000){
+            if (count > 20000) {
                 count = 0;
                 System.out.println("Pending count: " + this.pending.size() + ", total: " + this.total);
             }
             Thread.yield();
         }
 
         public void ack(Object msgId) {
-//            System.out.println("ACK");
+            //            System.out.println("ACK");
```

Comment:
Should we just remove this

Reply:
This is attempt to only fix format to allow for checkstyle to pass it. I did not go into each comment for its validity for deletion.